nbconvert/exporters/export.py

```diff
@@ -31,9 +32,14 @@
 #-----------------------------------------------------------------------------
 
 def DocDecorator(f):
-    
+    """
+    Deprecated since version 5.0.
```

Comment:
I don't think this needs a deprecation cycle - it's not meant to be public API, and it's [not documented](http://nbconvert.readthedocs.io/en/latest/api/exporters.html). It's only used in one place now, I think, so let's go ahead and remove it.